docs/reference/pip_download.rst

```diff
@@ -0,0 +1,41 @@
+
+.. _`pip download`:
+
+pip download
+------------
+
+.. contents::
+
+Usage
+*****
+
+.. pip-command-usage:: download
+
+``pip download`` is used just like ``pip install --download`` was used in the past. The same rules apply to this command: it should be very comfortable to use.
```

Comment:
This should probably be documented on it's own and not in relation to ``pip install --download`` which will be deprecated and removed.